### PR TITLE
HW01 by Oleg Kazak

### PR DESCRIPTION
**Pull request by Aleh13**
Task 1 completed

### CODE REVIEW

**Comment by vladhilko** [comment]
Could you use a better name and description for this pull request? 🤔

File: HW01/oleg_kazak/oleg_kazak_hw_01_t_01.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+print 'Enter the value of the hours hand: '
+hours = gets.to_i
+print 'Enter the value of the minutes hand: '
+minutes = gets.to_i
+def angle_time(hours, minutes)
+  # Degrees the hour hand passes in a minute
```
Comment (vladhilko) [inline HW01/oleg_kazak/oleg_kazak_hw_01_t_01.rb]: Nice explanation 👍

File: HW01/oleg_kazak/oleg_kazak_hw_01_t_01.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  hour_hand = 1 / (12 * 60.0) * 360
+  # Degrees the minute hand for a minute
+  minutes_hand = 360 * 1 / 60
```
Comment (vladhilko) [inline HW01/oleg_kazak/oleg_kazak_hw_01_t_01.rb]: We can move these variables to `CONSTANTS`

File: HW01/oleg_kazak/oleg_kazak_hw_01_t_01.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  # Clockwise angle
+  hour_angle = hour_hand * (minutes + hours * 60)
```
Comment (vladhilko) [inline HW01/oleg_kazak/oleg_kazak_hw_01_t_01.rb]: You don't need this comment if you think that the name is not clear just rename the variable, for example:
```suggestion
  clockwise_angle= hour_hand * (minutes + hours * 60)
```

File: HW01/oleg_kazak/oleg_kazak_hw_01_t_01.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  # Minute hand angle
+  minutes_angle = minutes_hand * minutes
```
Comment (vladhilko) [inline HW01/oleg_kazak/oleg_kazak_hw_01_t_01.rb]: The same
```suggestion
  minute_hand_angle = minutes_hand * minutes
```

File: HW01/oleg_kazak/oleg_kazak_hw_01_t_01.rb (hunk 1 above)
Comment (Aleh13) [reply HW01/oleg_kazak/oleg_kazak_hw_01_t_01.rb]: :)

File: HW01/oleg_kazak/oleg_kazak_hw_01_t_01.rb (hunk 3 above)
Comment (Aleh13) [reply HW01/oleg_kazak/oleg_kazak_hw_01_t_01.rb]: Changed the name of variables

**Comment by Aleh13** [comment]
> branches

File: HW01/oleg_kazak/oleg_kazak_hw_01_t_01.rb (hunk 2 above)
Comment (Aleh13) [reply HW01/oleg_kazak/oleg_kazak_hw_01_t_01.rb]: Constants Introduced

File: HW01/oleg_kazak/oleg_kazak_hw_01_t_01.rb (hunk 4 above)
Comment (Aleh13) [reply HW01/oleg_kazak/oleg_kazak_hw_01_t_01.rb]: Changed the name of variables

**Comment by vladhilko** [comment]
Why do you close this pr? 🤔

File: HW01/oleg_kazak/oleg_kazak_hw_01_t_01.rb (hunk 2 above)
Comment (vladhilko) [reply HW01/oleg_kazak/oleg_kazak_hw_01_t_01.rb]: I mean
```ruby 
HOUR_HAND = 1 / (12 * 60.0) * 360
# Degrees the minute hand for a minute
MINUTES_HAND = 360 * 1 / 60

def angle_time(hours, minutes)
  ...
end
```

Comment (Aleh13) [reply HW01/oleg_kazak/oleg_kazak_hw_01_t_01.rb]: fixed

Comment (vladhilko) [reply HW01/oleg_kazak/oleg_kazak_hw_01_t_01.rb]: `CONSTANS` shouldn't be changeable 😅 I asked to move only your calculation with numbers

**Comment by vladhilko** [comment]
Please, update title and description and I will merge

**Comment by Aleh13** [comment]
'Why did you close this PR?' - it was a coincidence

**Pull request merged**